core/src/main/java/jenkins/model/lazy/LazyBuildMixIn.java

```diff
@@ -179,6 +179,7 @@ public final synchronized RunT newBuild() throws IOException {
         } catch (InstantiationException e) {
             throw new Error(e);
         } catch (IllegalAccessException e) {
+            LOGGER.log(Level.WARNING, String.format("A new build could not be created in job %s", asJob().getFullName()), e);
```

Comment:
And possibly something for the `InvocationTargetException`, but likely more in `handleInvocationTargetException()` method itself.